readthedocs/oauth/migrations/0012_create_new_table_for_remote_repository_normalization.py

```diff
@@ -0,0 +1,73 @@
+                ('default_branch', models.CharField(blank=True, max_length=150, null=True, verbose_name='Default branch of the repository')),
+                ('remote_id', models.CharField(db_index=True, max_length=128)),
+                ('vcs_provider', models.CharField(choices=[('github', 'GitHub'), ('gitlab', 'GitLab'), ('bitbucket', 'Bitbucket')], max_length=32, verbose_name='VCS provider')),
+                ('organization', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='repositories', to='oauth.RemoteOrganization', verbose_name='Organization')),
+                ('project', models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='remote_repository', to='projects.Project')),
+            ],
+            options={
+                'verbose_name_plural': 'remote repositories',
+                'db_table': 'oauth_remoterepository_2020',
+                'ordering': ['organization__name', 'name'],
+            },
+        ),
+        migrations.CreateModel(
+            name='RemoteRepositoryRelation',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True, verbose_name='created')),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, verbose_name='modified')),
+                ('admin', models.BooleanField(default=False, verbose_name='Has admin privilege')),
+                ('json', jsonfield.fields.JSONField(verbose_name='Serialized API response')),
+                ('account', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='remote_repository_relations', to='socialaccount.SocialAccount', verbose_name='Connected account')),
+                ('remoterepository', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='remote_repository_relations', to='oauth.RemoteRepository')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='remote_repository_relations', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'unique_together': {('remoterepository', 'account')},
+            },
+        ),
+        migrations.AddField(
+            model_name='remoterepository',
+            name='users',
+            field=models.ManyToManyField(related_name='oauth_repositories', through='oauth.RemoteRepositoryRelation', to=settings.AUTH_USER_MODEL, verbose_name='Users'),
+        ),
+        migrations.AlterUniqueTogether(
+            name='remoterepository',
+            unique_together={('remote_id', 'vcs_provider')},
+        ),
```

Comment:
Why this one is not in the `options` where the `RemoteRepository` model is created (some lines above)?

Reply:
I'm not quite sure either, I auto-generated the migration with django's `manage.py makemigrations` management command.

Reply:
Ah, OK. I thought you wrote this file manually. Forget about my comment, then.